### PR TITLE
engine: add 24.0.0 release notes

### PR DESCRIPTION
**Pull request by neersighted**

### CODE REVIEW

**Review by vvoland — DISMISSED** [review]

**Review by dvdksn — APPROVED** [review]

File: engine/release-notes/24.0.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* libnetwork: Remove usage of the `xt_u32` kernel module from encrypted Swarm overlay networks. [moby/moby#45281](https://github.com/moby/moby/pull/45281)
+* Remove support for buildkit's deprecated `buildinfo` in favor of standard provenance attestations. [moby/moby#45097](https://github.com/moby/moby/pull/45097)
+* Remove the deprecated AUFS and legacy `overlay` storage drivers. [moby/moby#45342](https://github.com/moby/moby/pull/45342), [moby/moby#](https://github.com/moby/moby/pull/45359)
+* Remove the deprecated `overlay2.override_kernel_check` storage driver option. [moby/moby#45368](https://github.com/moby/moby/pull/45368)
+* Remove workarounds for obsolete versions of `apparmor_parser` from the AppArmor profiles. [moby/moby#45500](https://github.com/moby/moby/pull/45500)
+* API:  `GET /images/json` no longer represents empty RepoTags and RepoDiegsts as`<none>:<none>`/`<none>@<none>`. Empty arrays are be returned instead on API >= 1.43. [moby/moby#45068](https://github.com/moby/moby/pull/45068)
+
+### Deprecated
+* Deprecate the `--oom-score-adjust` daemon option. [moby/moby#45315](https://github.com/moby/moby/pull/45315)
```
Comment (dvdksn) [inline engine/release-notes/24.0.md:56]: Add newline

```suggestion

* Deprecate the `--oom-score-adjust` daemon option. [moby/moby#45315](https://github.com/moby/moby/pull/45315)
```

File: engine/release-notes/24.0.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* Fix numerous bugs in the embedded DNS resolver implementation used by user-defined networks. [moby/moby#44664](https://github.com/moby/moby/pull/44664)
+* Add `execDuration` field to the map of event attributes. [moby/moby#45494](https://github.com/moby/moby/pull/45494)
+* Swarm-level networks can now be created with the Windows `internal`, `l2bridge`, and `nat` drivers. [moby/swarmkit#3121](https://github.com/moby/swarmkit/pull/3121), [moby/moby#45291](https://github.com/moby/moby/pull/45291)
+
+### Packaging updates
+
+* Update Go to `1.20.4`. [docker/cli#4253](https://github.com/docker/cli/pull/4253), [moby/moby#45456](https://github.com/moby/moby/pull/45456), [docker/docker-ce-packaging#888](https://github.com/docker/docker-ce-packaging/pull/888)
+* Update `containerd` to [`v1.7.1`](https://github.com/containerd/containerd/releases/tag/v1.7.1). [moby/moby#45537](https://github.com/moby/moby/pull/45537)
+* Update `buildkit` to [`v0.11.4`](https://github.com/moby/buildkit/releases/v0.11.4). [moby/moby#45109](https://github.com/moby/moby/pull/45109)
```
Comment (crazy-max) [inline engine/release-notes/24.0.md:73]: Should be 0.11.6: https://github.com/moby/moby/blob/1331b8c39a601c77d678f3e03f3f39cb07aa5f76/vendor.mod#L59
* https://github.com/moby/moby/pull/45214
* https://github.com/moby/moby/pull/45367

**Review by dvdksn — APPROVED** [review]

File: engine/release-notes/24.0.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 24.0.0 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A24.0.0)
+- [moby/moby, 24.0.0 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A24.0.0)
+
+### New
+
+* Introduce experimental support for containerd as the content store (replacing the existing storage drivers). [moby/moby#43735](https://github.com/moby/moby/pull/43735), [other moby/moby pull requests](https://github.com/moby/moby/pulls?q=is%3Apr+is%3Amerged+milestone%3A24.0.0+-label%3Aprocess%2Fcherry-picked+label%3Acontainerd-integration+)
+* The `--host` CLI flag now supports a path component in a `ssh://` host address, allowing use of an alternate socket path without configration on the remote host. [docker/cli#4073](https://github.com/docker/cli/pull/4073)
```
Comment (crazy-max) [inline engine/release-notes/24.0.md:34]: Small typo `/configration/configuration`

File: engine/release-notes/24.0.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+* The `docker info` CLI command now reports a version and platform field. [docker/cli#4180](https://github.com/docker/cli/pull/4180)
+* Introduce the daemon flag `--default-network-opt` to configure options for newly created networks. [moby/moby#43197](https://github.com/moby/moby/pull/43197)
+* Restrict access to `AF_VSOCK` in the `socket(2)` family of syscalls in the default seccomp profile. [moby/moby#44562](https://github.com/moby/moby/pull/44562)
+* Introduce support for setting OCI runtime annotations on containers. [docker/cli#45025](https://github.com/docker/cli/pull/4156), [moby/moby#45025](https://github.com/moby/moby/pull/45025)
+* Alternative runtimes can now be configured in daemon.json, enabling runtime names to be aliased and options to be passed. [moby/moby#45032](https://github.com/moby/moby/pull/45032)
```
Comment (crazy-max) [inline engine/release-notes/24.0.md:39]: nit: `daemon.json` as inline code

File: engine/release-notes/24.0.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+* Fix numerous bugs in the embedded DNS resolver implementation used by user-defined networks. [moby/moby#44664](https://github.com/moby/moby/pull/44664)
+* Add `execDuration` field to the map of event attributes. [moby/moby#45494](https://github.com/moby/moby/pull/45494)
+* Swarm-level networks can now be created with the Windows `internal`, `l2bridge`, and `nat` drivers. [moby/swarmkit#3121](https://github.com/moby/swarmkit/pull/3121), [moby/moby#45291](https://github.com/moby/moby/pull/45291)
+
+### Packaging updates
+
+* Update Go to `1.20.4`. [docker/cli#4253](https://github.com/docker/cli/pull/4253), [moby/moby#45456](https://github.com/moby/moby/pull/45456), [docker/docker-ce-packaging#888](https://github.com/docker/docker-ce-packaging/pull/888)
+* Update `containerd` to [`v1.7.1`](https://github.com/containerd/containerd/releases/tag/v1.7.1). [moby/moby#45537](https://github.com/moby/moby/pull/45537)
+* Update `buildkit` to [`v0.11.6`](https://github.com/moby/buildkit/releases/v0.11.4). [moby/moby#45367](https://github.com/moby/moby/pull/45367)
```
Comment (crazy-max) [inline engine/release-notes/24.0.md:74]: Link looks wrong, should be https://github.com/moby/buildkit/releases/v0.11.6

**Pull request merged**